lib/openstudio-standards/constructions/create.rb

```diff
@@ -5,15 +5,31 @@ module Constructions
     # Methods to create Constructions
 
     # This will create a deep copy of the construction, meaning it will clone and create new material objects as well
+    # This function will clone
     #
     # @param construction [OpenStudio::Model::Construction] OpenStudio Construction object object
     # @return [OpenStudio::Model::Construction] New OpenStudio Construction object
     def self.construction_deep_copy(construction)
```

Comment:
@mdahlhausen I modified this function to include deep copy for f factor ground floor construction, c factor underground wall construction.
I wonder if you prefer to keep a strict typing which means the return is limited to one type? If so, I can create separate deep clone functions.
Also, `OS_Construction_AirBoundary` is not included in this function, do you think it is necessary to include it in here too?

Reply:
A more general deep copy function is appropriate. Thanks.

I'm getting some new errors line 26.

Reply:
> A more general deep copy function is appropriate. Thanks.
> 
> I'm getting some new errors line 26.

@mdahlhausen that is because the AirBoundary construction is not included in this function. I can include the AirBoundary type to here as well.